torch/profiler/profiler.py

```diff
@@ -44,6 +52,22 @@ def supported_activities():
     return torch.autograd._supported_activities()
 
 
+def _optimizer_post_hook(optimizer, args, kwargs):
+    KinetoStepTracker.increment_step("Optimizer")
+
+
+def _profile_using_dynolog(override: bool = False):
+    """
+    To enable tracing via dynolog we register a global optimizer step post
+    hook. Requires the 'KINETO_USE_DAEMON' environment variable to be set.
+    """
+    global _dynolog_handle
+    if _dynolog_handle is not None and os.environ.get("KINETO_USE_DAEMON", None) is not None or override is True:
+        _dynolog_handle = True
+        _ = register_optimizer_step_post_hook(_optimizer_post_hook)
+        logger.info("Registered optimizer step post hook")
```

Comment:
Is this just debug code?

Reply:
In my opinion, this should be an info level log statement instead of debug level. This clearly informs the user that a post hook has been registered. If you think this should be a debug level statement, then I can change it.

Reply:
I general, I don't think we should log at all when user calls our APIs.

Reply:
i'll change to a debug level log statement.